src/index.jsx

```diff
@@ -3,7 +3,7 @@ import { createRoot } from 'react-dom/client';
 import { MainView } from './components/main-view/main-view'; 
 import 'bootstrap/dist/css/bootstrap.min.css'; // Added import for Bootstrap styles
```

Comment:
You have this line, but before this works you must first install `bootstrap`. Do that and let me know how you did it.

Reply:
i used command promt